### PR TITLE
add vm.tiktok.com for supported urls

### PR DESCRIPTION
**Pull request by 532910**

### CODE REVIEW

**Comment by 532910** [comment]
Could or should `*.tiktok.com` be added?

**Comment by nicholascioli** [comment]
I don't know why your checks are failing, considering you didn't change anything that would mess with the build. Could you rebase this on the dev branch? I try to leave merging into master until after I've better tested the features.

As for adding *.tiktok.com, I don't know if we should because the app does not yet support all of TikTok's features. We are still missing LiveStreams, messaging, etc. So I think we should keep the scope narrow until we add feature parity.

Looks good otherwise!

File: android/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -51,6 +51,9 @@
               <data
                 android:scheme="https"
                 android:host="tiktok.com" />
+              <data
+                android:scheme="https"
+                android:host="vm.tiktok.com" />
```
Comment (nicholascioli) [inline android/app/src/main/AndroidManifest.xml]: This looks good. What links include the vm subdomain?

File: android/app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -59,4 +62,4 @@
             android:name="flutterEmbedding"
             android:value="2" />
     </application>
-</manifest>
\ No newline at end of file
+</manifest>
```
Comment (nicholascioli) [inline android/app/src/main/AndroidManifest.xml]: Looks good

**Review by nicholascioli — CHANGES_REQUESTED** [review]
Just rebase and it should be good.

**Comment by 532910** [comment]
> Just rebase and it should be good.

Just rebased, but checks still fails.

File: android/app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (532910) [reply android/app/src/main/AndroidManifest.xml]: Direct links to video.

Like https://vm.tiktok.com/ZSpdohha/ for example that is expanded to https://www.tiktok.com/@johnvanb/video/6870536778642771201 with some garbage at the end.

To create a vm link just share a video from TikTokLite.

**Comment by 532910** [comment]
I loose it, sorry ):

**Pull request closed without merging**